cmssw-queue-override.file

```diff
@@ -36,7 +36,7 @@ Source20: CXXModules.mk
     echo "<flags LCGDICT_PACKAGE=\\"$p\\"/>"  >> config/BuildFile.xml \
   done
   echo "<use name=\\"CondFormats/L1TObjects\\"/>"	> bf
-  cat CondFormats/RPCObjects/BuildFile > bf
-  mv bf CondFormats/RPCObjects/BuildFile
+  cat src/CondFormats/RPCObjects/BuildFile > bf
+  mv bf src/CondFormats/RPCObjects/BuildFile
```

Comment:
you are missing `.xml` extension here. it should be  `src/CondFormats/RPCObjects/BuildFile.xml`.
By the way you need to check if `src/CondFormats/RPCObjects/BuildFile.xml` exists (note that for patch builds this might be missing). Also such changes are not going to work in dev area e.g. when one addpkg `CondFormats/RPCObjects`